### PR TITLE
PAVP: set vpp parameters if layer is protected

### PR DESCRIPTION
**Pull request by js0701**
Also fix logic error related with SetProtected

Jira: None
Test: Android UI normal
Signed-off-by: Lin Johnson <johnson.lin@intel.com>

### CODE REVIEW

File: common/compositor/va/varenderer.cpp [hunk 1]
```diff
@@ -295,6 +296,11 @@ bool VARenderer::Draw(const MediaState& state, NativeSurface* surface) {
   pipe_param.output_region = &output_region;
   pipe_param.output_color_standard = VAProcColorStandardBT601;
 
+  if (state.layer_->IsProtected()) {
+    /*Turn on protected flag*/
+    pipe_param.input_surface_flag = 1;
```
Comment (stripes416) [inline common/compositor/va/varenderer.cpp]: Travis if failing with this error:  
```
compositor/va/varenderer.cpp: In member function virtual bool hwcomposer::VARenderer::Draw(const hwcomposer:   :MediaState&, hwcomposer::NativeSurface*):
compositor/va/varenderer.cpp:301:16: error: VAProcPipelineParameterBuffer has no member named input_su   rface_flag
 pipe_param.input_surface_flag = 1;
                 ^
make[3]: *** [compositor/va/varenderer.lo] Error 1
make[2]: *** [all-recursive] Error 1
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
```

Comment (js0701) [reply common/compositor/va/varenderer.cpp]: I have compiled this in OneA. But it failed on Linux

Comment (js0701) [reply common/compositor/va/varenderer.cpp]: Updated to add MACRO to control this

Comment (js0701) [reply common/compositor/va/varenderer.cpp]: Will merge this as Tong Liang said he has verified this patch can work

**Comment by js0701** [comment]
closed as merged

**Pull request closed without merging**